### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by dnadeau4**

### CODE REVIEW

**Comment by dnadeau4** [comment]
I am proposing these standard issue template to facilitate understanding and reproduction of the bug.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
```
Comment (doutriaux1) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Do we need this?

Comment (doutriaux1) [reply .github/ISSUE_TEMPLATE/bug_report.md]: what about a `conda list` section?

like this:

<details><summary><code>`conda list --show-channel-urls`</code></summary><p>


```

```
</p></details>

**Review by doutriaux1 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (dnadeau4) [reply .github/ISSUE_TEMPLATE/bug_report.md]: People make bug reports that I cannot reproduce.  In numpy when I make a bug report, I make sure I have the code that fails, the error and I can easily reproduce it.

Comment (doutriaux1) [reply .github/ISSUE_TEMPLATE/bug_report.md]: @dnadeau4 I meant the smartphone section.

Comment (dnadeau4) [reply .github/ISSUE_TEMPLATE/bug_report.md]: These are default template.  I can delete this section.

**Comment by doutriaux1** [comment]
that would be nice. Also adding the conda list section (folded) would be great. see: https://github.com/conda/conda/issues/new?template=Bug_report.md for more possibilities to add

**Review by doutriaux1 — APPROVED** [review]

**Pull request merged**